packages/drivers/odsp-driver/src/odspDocumentStorageManager.ts

```diff
@@ -670,6 +671,7 @@ export class OdspDocumentStorageService implements IDocumentStorageService {
             }
 
             const clientTime = networkTime ? overallTime - networkTime : undefined;
+            const isAfd = msEdge !== undefined;
```

Comment:
@sunnyhuang2008 As I checked https://developer.mozilla.org/en-US/docs/Web/API/Headers/get the return type of get is null if header not present. Can you please make change to it and double check?